hikari/api/shard.py

```diff
@@ -374,7 +374,7 @@ async def request_guild_members(
             If not `builtins.None`, request the members which username starts with the string.
         limit: builtins.int
             Maximum number of members to send matching the query.
-        user_ids: hikari.undefined.UndefinedOr[typing.Sequence[hikari.snowflakes.SnowflakeishOr[hikari.users.User]]]
+        users: hikari.undefined.UndefinedOr[typing.Sequence[hikari.snowflakes.SnowflakeishOr[hikari.users.User]]]
```

Comment:
This should also be marked as a breaking change along with all the changes to the chunker